### DISCUSSION TITLE
Jaume Amores - Learning visually

### DISCUSSION THREAD

**Discussion by giscus** [beatrizmilz/blog-en#32] [General]
# Jaume Amores - Learning visually

Short notes on learning by practice and visualization.

https://jaumeamoresds.github.io/home/posts/education/learning_visually.html

**Comment by guruglen** [comment]
This looks very interesting Jaume. I agree that with our burgeoning understanding of learning, we are just beginning to see what technology can do in education. I will be using your ideas here to further focus my learning in this area. Thanks for the sign posts.

**Reply by JaumeAmoresDS** [reply]
Thanks @guruglen. I will include additional references. Sorry for the very late reply... I missed the notification.
